src/docs/product/releases/index.mdx

```diff
@@ -8,9 +8,9 @@ redirect_from:
 description: "Learn how to provide information to Sentry about your releases to determine regressions and resolve issues quickly."
 ---
 
-A _release_ is a version of your code deployed to an environment. When you notify Sentry about a release, you can easily identify new issues, regressions, whether an issue is resolved in the next release, and apply [source maps](/platforms/javascript/sourcemaps/).
+A _release_ is a version of your code deployed to an environment. When you notify Sentry about a <SandboxLink scenario="oneRelease" projectSlug="react">release</SandboxLink>, you can easily identify new issues, regressions, whether an issue is resolved in the next release, and apply [source maps](/platforms/javascript/sourcemaps/).
 
-Releases also offer significant additional features when you fully [configure your SDK](/platform-redirect/?next=/configuration/releases/):
+<SandboxLink scenario="releases" projectSlug="react">Releases</SandboxLink> also offer significant additional features when you fully [configure your SDK](/platform-redirect/?next=/configuration/releases/):
```

Comment:
The platform redirect link is breaking and I think it's because you're using HTML style tagging in the same line of code. Can the sandbox link be done in markdown instead? Please try committing this change and checking if the preview works after that. 

```suggestion
[Releases](https://try.sentry-demo.com/demo/start/?scenario=releases&projectSlug=react&source=docs) also offer significant additional features when you fully [configure your SDK](/platform-redirect/?next=/configuration/releases/):
```

Reply:
I wouldn't recommend using sandbox links directly as they may change. I was able to get this to work by doing this:
```
<SandboxLink scenario="releases" projectSlug="react">Releases</SandboxLink> also offer significant
 additional features when you fully <a href="/platform-redirect/?next=/configuration/releases/">configure your SDK</a>:
```

I will look into why markdown links didn't render.

Reply:
I'm pretty sure it's the same underlying behaviour that causes markdown to not render properly in a note if you don't add lines between the HTML tags of the note and the markdown. If the HTML tag is the first thing in the line, and there's no break, Gatsby doesn't seem to read the markdown.